README.md

```diff
@@ -1,63 +1,99 @@
+
+[Quality filtering](docs/user-guide/QualityFiltering.rst)
+
+- Multilingual heuristic-based filtering
+- Classifier-based filtering via [fastText](https://fasttext.cc/)
+
+[Document-level deduplication](docs/user-guide/GpuDeduplication.rst)
+
+- Both exact and fuzzy deduplication are accelerated using cuDF and Dask
+- For fuzzy deduplication, our implementation follows the method described in [Microsoft Turing NLG 530B](https://arxiv.org/abs/2201.11990)
+
+[Multilingual downstream-task decontamination](docs/user-guide/TaskDecontamination.rst)
+
+- Our implementation follows the approach of [OpenAI GPT3](https://arxiv.org/pdf/2005.14165.pdf) and [Microsoft Turing NLG 530B](https://arxiv.org/abs/2201.11990)
+
+[Distributed data classification](docs/user-guide/DistributedDataClassification.rst)
+
+- Multi-node, multi-GPU classifier inference
+- Provides sophisticated domain and quality classification
+- Flexible interface for extending to your own classifier network
+
+[Personal identifiable information (PII) redaction](docs/user-guide/PersonalIdentifiableInformationIdentificationAndRemoval.rst)
+
+- Identification tools for removing addresses, credit card numbers, social security numbers, and more
+
+These modules offer flexibility and permit reordering, with only a few exceptions. In addition, the [NeMo Framework Launcher](https://github.com/NVIDIA/NeMo-Megatron-Launcher) provides pre-built pipelines that can serve as a foundation for your customization use cases.
+
+## Resources
+
 - [Documentation](docs/)
 - [Examples](examples/)
-- [Module Ablation and Compute Performance](#module-ablation-and-compute-performance)
+- [Tutorials](tutorials/)
 
-## Installation
+## Get Started
 
-NeMo Curator currently requires Python 3.10 and the GPU accelerated modules require CUDA 12 or above installed in order to be used.
+This section explains how to install NeMo Curator and use the Python library, Python modules, and CLI scripts. It also includes a list of tutorials to help you get started right away. Finally, this section explains how to use the NeMo Framework Launcher as an alternative method for interfacing with NeMo Curator.
 
-NeMo Curator can be installed manually by cloning the repository and installing as follows -
+## Requirements
 
-For CPU only modules:
-```
-pip install .
-```
+Before installing NeMo Curator, ensure that the following requirements are met:
 
-For CPU + CUDA accelerated modules
-```
-pip install --extra-index-url https://pypi.nvidia.com ".[cuda12x]"
-```
+- Python 3.10 (or above)
+- CUDA 12 (or above)
+- NVIDIA GPU (optional)
+
+## Install NeMo Curator
+
+Two options are available for installing NeMo Curator. You  can install it from the repository or through the NeMo Framework container.
+
+### Install from the Repository
+
+1. Clone the NeMo Curator repository in GitHub.
+
+2. Install the modules that you need.
+
+    To install the CPU-only modules:
+
+    ```
+    pip install
```

Comment:
This command should have a dot in the end:

```
pip install .
```

Reply:
Added dot